strabo/static/js/popup.js

```diff
@@ -88,7 +88,7 @@ while making sure that photoswipe is not opened twice.
 The delay is the second argument to window.setTimeout in units of milliseconds. This can be played with,
 but very small values like 1 do not work very well.
 */
-function set_flickety_click(){
+function set_flickity_click(){
```

Comment:
I don't really know how to describe the problem much better, so the change has my best attempt. And I don't really know why the solution works, I just felt like adding a delay would help, and it did. One of those instincts you acquire by diving into the horrors of event processing.